src/libs/actions/Report.ts

```diff
@@ -892,11 +892,11 @@ function navigateToAndOpenReport(
     avatarUri?: string,
     avatarFile?: File | CustomRNImageManipulatorResult | undefined,
     optimisticReportID?: string,
+    isGroupChat?: boolean,
```

Comment:
@abzokhattab Should we use false as the default value for `isGroupChat`?

Reply:
Both would work .. but I agree we can explicitly make the default as `false` to be more descriptive and clear.